Readme.md

```diff
@@ -521,7 +525,12 @@ $ custom --list x,y,z
 
 ## Commands
 
-You can specify (sub)commands using `.command()` or `.addCommand()`. There are two ways these can be implemented: using an action handler attached to the command, or as a stand-alone executable file (described in more detail later). The subcommands may be nested ([example](./examples/nestedCommands.js)).
+You can specify (sub)commands using `.command()` or `.addCommand()`. There are two ways these can be implemented:
+
+1. Using an action handler attached to the command; or,
+2. as a stand-alone executable file. (More detail about this later.)
```

Comment:
I don't think this expansion is enough of an improvement over the concise statement.

Reply:
K.